src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -1,5 +1,10 @@
 package bio.terra.service.snapshotbuilder;
 
+import static bio.terra.service.snapshotbuilder.utils.QueryBuilderConstants.CONCEPT_ID;
```

Comment:
In general, static imports are bad as they make it harder to see where a name is coming from. We make an exception for the mock and hamcrest APIs used in tests, but other cases shouldn't use them.